testnets/okp4testnet/assetlist.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "okp4testnet",
+    "assets": [
+      {
+        "description": "The native token of ØKP4.",
+        "denom_units": [
+          {
+            "denom": "uknow",
+            "exponent": 0
+          },
+          {
+            "denom": "know",
+            "exponent": 6
+          }
+        ],
+        "base": "uknow",
+        "name": "OKP4 Network",
+        "display": "know",
+        "symbol": "KNOW",
+        "logo_URIs": {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/okp4testnet/images/okp4.svg"
```

Comment:
I won't require it, but any chance you have an image file with square aspect ratio?

Reply:
Yes I've updated the logo with a square one.